src/test/resources/application.yml

```diff
@@ -16,10 +16,11 @@ spring:
 logging:
   level:
     root: ${ROOT_LOG_LEVEL}
+    org.postgresql: DEBUG
```

Comment:
Just wanted to point this out to make sure you want to keep this set as DEBUG.  Another thing we could do is add this as an environment variable like the line above: `ROOT_LOG_LEVEL` .  That way, we can adjust the postgres logging level from the aws management console on the fly without having to submit PRs.

Reply:
This is in the /test/.../application.yml, so its not the prod one.  The prod one looks ok.  As a reference, this matches the settings in the same file of the discrete loader.

Reply:
Ah, roger that.  I didn't see the `/test` - in that case I think this is a really good default to have for devs running the project locally.